certified-connectors/OzyApprovals/apiDefinition.swagger.json

```diff
@@ -1 +1,1285 @@
+                      "description": "Request Platform"
+                    },
+                    "process_name": {
+                      "type": "string",
+                      "description": "Process Name"
+                    },
+                    "process_description": {
+                      "type": "string",
+                      "description": "Process Description"
+                    },
+                    "process_id": {
+                      "type": "string",
+                      "description": "Process ID"
+                    },
+                    "request_id": {
+                      "type": "string",
+                      "description": "Request ID"
+                    },
+                    "request_data": {
+                      "type": "object"
+                    },
+                    "approval_stages": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "id": {
+                            "type": "string",
+                            "description": "Stage ID"
+                          },
+                          "names": {
+                            "type": "string",
+                            "description": "Stage Names"
+                          },
+                          "description": {
+                            "type": "string",
+                            "description": "Stage Description"
+                          },
+                          "deleted": {
+                            "type": "string",
+                            "description": "Is the stage deleted?"
+                          }
+                        }
+                      }
+                    }
+                  },
+                  "description": "msg"
+                }
+              }
+            },
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Find an Approval Request",
+        "operationId": "FindApprovalProcessRequest",
+        "description": "Search for an approval request by ID",
+        "parameters": [
+          {
+            "name": "approval_process_request_id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-dynamic-values": {
+              "operationId": "GetApprovalProcessRequests",
+              "value-path": "id",
+              "value-collection": "msg",
+              "value-title": "name"
+            },
+            "description": "Request ID",
+            "x-ms-summary": "Request ID"
+          }
+        ]
+      }
+    },
+    "/retrieve_approval_stage_request/{approval_stage_request_id}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
Please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.